src/index.html

```diff
@@ -11,6 +11,36 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form action="https://www.google.com/search" 
+      method="get"  
+      class="search_form" 
+      data-qa="big">
+
+      <label for="big_search" class="search_label">
+        <input class="search_input" 
+        type="text" 
+        placeholder="Try &quot;Los Angeles&quot;" 
+        id="big_search"
+        data-qa="keypress">
+      </label>
```

Comment:
```suggestion
        <input
          class="search_input" 
          type="text" 
          placeholder="Try &quot;Los Angeles&quot;" 
          id="big_search"
          data-qa="keypress"
        >
      </label>
```

Reply:
no need for='''' and id="" when label includes the input, but when input is tot inside you need this connection)